debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1133,6 +1143,10 @@ protected SourceInfoStructMaker<? extends AbstractSourceInfo> getSourceInfoStruc
         return getSourceInfoStructMaker(SOURCE_INFO_STRUCT_MAKER, Module.name(), Module.version(), this);
     }
 
+    protected int getGrowingWalWarningLogInterval() {
+        return getConfig().getInteger(GROWING_WAL_WARNING_LOG_INTERVAL);
```

Comment:
This call will be on critical path so it is better to create a standalone attribute, initialize it in constructor and just use it here.